### DISCUSSION TITLE
Windows NTS Phalcon DLL 4.1.3  ready to download

### DISCUSSION THREAD

**Discussion by MarceloLemosbr** [phalcon/cphalcon#16220] [Q&A]
Hello all, 

is there a link to get the DLL 4.1.3 ready to use in windows with PHP NTS?

**Comment by niden** [comment]
There are none. 4.1.3 was a patch release for a few issues. We really did not have a steady workflow where to create DLLs on demand (we had to wait for 3 hours for them to be built), so never did them with that version - since it is not supported.

If possible, try to upgrade to v5
